tutor/templates/apps/openedx/settings/partials/common_cms.py

```diff
@@ -21,9 +21,9 @@
 FRONTEND_REGISTER_URL = LMS_ROOT_URL + '/register'
 
 # Create folders if necessary
-for folder in [LOG_DIR, MEDIA_ROOT, STATIC_ROOT_BASE, ORA2_FILEUPLOAD_ROOT]:
+for folder in [LOG_DIR, MEDIA_ROOT, STATIC_ROOT, ORA2_FILEUPLOAD_ROOT]:
     if not os.path.exists(folder):
-        os.makedirs(folder, exist_ok=True)
+        os.makedirs(folder, exist_ok=True, parents=True)
```

Comment:
Note for reviewers: We had to get rid of STATIC_ROOT_BASE, because STATIC_ROOT now comes from the environment. So, instead of creating STATIC_ROOT_BASE (`/openedx/staticfiles`), we are creating STATIC_ROOT (`/openedx/staticfiles/studio`). This works fine, as long as we supply `parents=True`, which tells `os.makedirs` that it can create multiple levels of directories.